podcasts/UIScreen+Sizes.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension UIScreen {
+    static var isSmallScreen: Bool {
+       UIScreen.main.bounds.height < 667
```

Comment:
On the iPhone SE this returns false because its height is 667. 

```suggestion
       UIScreen.main.bounds.height <= 667
```